submissions/MosQuitO404/OOPJS_frogger/app.js

```diff
@@ -0,0 +1,131 @@
+// Enemies our player must avoid
+const Enemy = function() {
+    // Variables applied to each of our instances go here,
+    // we've provided one for you to get started
+    
+    // The image/sprite for our enemies, this uses
+    // a helper we've provided to easily load images
+    this.x = -95;
+    this.y = this.positionEnemyMaker();
+    this.speed = this.speedEnemyMaker(200, 450);
+    this.sprite = 'images/enemy-bug.png';
+};
+
+// Update the enemy's position, required method for game
+// Parameter: dt, a time delta between ticks
+Enemy.prototype.update = function(dt) {
+    // You should multiply any movement by the dt parameter
+    // which will ensure the game runs at the same speed for
+    // all computers.
+    if (this.x < 500) { 
```

Comment:
50, -95 and other numbers seem mysterious.
See section 3 of the requirements.

Reply:
Ok, resolved. It's canvas size :)